src/mozanalysis/experiment.py

```diff
@@ -137,7 +158,7 @@ def get_single_window_data(
 
         Args:
             bq_context (BigQueryContext): BigQuery configuration and client.
-            metric_list (list of mozanalysis.metric.Metric): The metrics
+            metric_list (list of mozanalysis.metric.Metric or str): The metrics
```

Comment:
Just confirming I understand correctly -- use of `mozanalysis.metric.Metric` here is deprecated, and we expect to use `str` (metric slug) going forward?

Reply:
That is mostly correct. `Metric` can still be used if, for example, a DS defines their own custom metric in a Jupyter notebook that uses mozanalysis. But for referencing existing metrics `str` is recommended